namer/namer.cc

```diff
@@ -629,12 +629,24 @@ class SymbolFinder {
         return foundDefs->addTypeMember(move(found));
     }
 
+    bool sendRecvIsT(const ast::Send &s) {
```

Comment:
I found this very similar function:

https://github.com/sorbet/sorbet/blob/92839969971586d2ef06fa7731ce8036092e7025/cfg/builder/builder_walk.cc#L109-L115

I renamed my `isT()` to the same name, and edited the parameter type to match.